app/src/main/java/org/systers/mentorship/view/activities/SendRequestActivity.kt

```diff
@@ -36,6 +43,52 @@ class SendRequestActivity: BaseActivity() {
         val currentUserId = getAuthTokenPayload().identity
         setObservables()
         populateView(otherUserName, otherUserId, currentUserId)
+
+
+        myCalendar = Calendar.getInstance()
+        var date : DatePickerDialog.OnDateSetListener = object : DatePickerDialog.OnDateSetListener{
+
+            override fun onDateSet(view: DatePicker?, year: Int, month: Int, dayOfMonth: Int) {
+
+                myCalendar.set(Calendar.YEAR , year)
+                myCalendar.set(Calendar.MONTH  , month)
+                myCalendar.set(Calendar.DAY_OF_MONTH  , dayOfMonth)
+
+                updateEditText()
+
+            }
+
+
+        }
+
+
+
+
+        calendar_icon.setOnClickListener {
+
+
+          DatePickerDialog(this , date ,
+                  myCalendar.get(Calendar.YEAR) ,
+                  myCalendar.get(Calendar.MONTH) ,
+                  myCalendar.get(Calendar.DAY_OF_MONTH)).show()
+      }
+
+
+
+    }
+
+    private fun updateEditText() {
+
+
+
+
+        var myFormat : String = "dd/MM/yyyy"
+        var sdf : SimpleDateFormat = SimpleDateFormat(myFormat , Locale.US)
```

Comment:
Instead of hardcoding the format, use directly `SEND_REQUEST_END_DATE_FORMAT` that is already used in this file. I think `SEND_REQUEST_END_DATE_FORMAT` = "dd/MM/yyyy" (confirm this first)